packages/fast-components-react-msft/src/action-trigger/action-trigger.tsx

```diff
@@ -8,6 +8,7 @@ import {
     ActionTriggerUnhandledProps,
 } from "./action-trigger.props";
 import { actionTriggerButtonOverrides } from "@microsoft/fast-components-styles-msft";
+import { isNullOrUndefined } from "util";
```

Comment:
I don't see this in `package.json` - what is this import?

Reply:
+1 it looks like this is getting auto-added from node, which we *do not* want.

Reply:
Yes, this was deprecated in node.
They suggested to use `value === undefined || value === null`
https://nodejs.org/api/util.html#util_util_isnullorundefined_object

I created a function, with the same name, in the utilities directory.
Also updated the reference in action-toggle and auto-suggest-option